ocamldoc/odoc_class.ml

```diff
@@ -192,7 +192,7 @@ let class_comments ?(trans=true) cl =
     (class_elements ~trans cl)
 
 
-(** Update the parameters text of a t_class, according to the cl_info field. *)
+(** Updates the parameters text of a t_class, according to the cl_info field. *)
```

Comment:
Inside this specific file, the infinitive is only used 1 time while the conjugate form (see 'Returns')
is used 10 times.
Note: otherwise, I prefer the infinitive form but making all the corrections would be a LOT of work.